content/rdi/architecture.md

```diff
@@ -9,59 +9,89 @@ headerRange: "[2]"
 aliases: 
 ---
 
-Redis Data Integration is a product for data ingestion from existing systems of record (mainly databases) into Redis Enterprise.
-You can think of Redis Data Integration as a streaming ELT process:
+Redis Data Integration (RDI) is a product that helps Redis Enterprise users ingest and export data in near real time. RDI provides the following:
 
-- Data is **E**xtracted from the source database using [Debezium](https://debezium.io/) connectors.
+- End to end solution - No need for additional tools and integrations
+- Capture Data Change (CDC) included
+- Covers most popular databases as sources and targets
+- Declarative data mapping and transformations - no custom code needed
+- Data delivery guaranteed (at least once)
+- Zero downtime day 2 operations (reconfigure & upgrade)
```

Comment:
ChatGPT suggestion, see if some of it is better:
RDI offers the following features:
- End-to-end solution: No additional tools or integrations required
- Includes Change Data Capture (CDC)
- Supports a wide range of popular databases as both sources and targets
- Provides declarative data mapping and transformations, eliminating the need for custom code
- Ensures data delivery with at least once guarantee
- Enables zero-downtime day 2 operations for reconfiguration and upgrades

Reply:
done